chart/values.yaml

```
@@ -0,0 +1,49 @@
+---
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+image:
+  pullPolicy: IfNotPresent
+  repository: ghcr.io/ministryofjustice/analytical-platform-ollamate
+  tag: 0.0.1
```

```suggestion
  tag: ""
```
Set to blank to use `appVersion`

---

✅